Library/High Tech/High Tech Equipment.eqp

```diff
@@ -10446,9 +10515,39 @@
 					"type": "dr_bonus",
 					"amount": 5,
 					"location": "foot"
+				},
+				{
+					"type": "dr_bonus",
+					"amount": -2,
+					"location": "skull",
+					"specialization": "not from front"
+				},
+				{
+					"type": "dr_bonus",
+					"amount": -2,
+					"location": "eye",
+					"specialization": "not from front"
+				},
+				{
+					"type": "dr_bonus",
+					"amount": -2,
+					"location": "face",
+					"specialization": "not from front"
+				},
+				{
+					"type": "dr_bonus",
+					"amount": -15,
+					"location": "torso",
+					"specialization": "not from front"
+				},
+				{
+					"type": "dr_bonus",
+					"amount": -15,
+					"location": "vitals",
+					"specialization": "not from front"
 				}
 			],
-			"notes": "Flexible.",
+			"notes": "Flexible. Torso DR20, head DR7, limb DR7: all from the front, 5 DR in all locations from all the other directions",
```

Comment:
Are these lengthy descriptions necessary, given the specialized DR entries?

Reply:
I was torn on this for some time, but now, yeah, I agree. I know at least 1 person who doesn't want to see lenghy notes.